cirq-core/cirq/ops/raw_types.py

```diff
@@ -664,6 +664,12 @@ def _channel_(self) -> Union[Tuple[np.ndarray], NotImplementedType]:
     def _measurement_key_(self) -> str:
         return protocols.measurement_key(self.sub_operation, NotImplemented)
 
+    def _is_measurement_(self) -> bool:
+        sub = getattr(self.sub_operation, "_is_measurement_", None)
+        if sub is not None:
+            return sub()
+        return NotImplemented  # coverage: ignore
```

Comment:
Could you add a test for this with a tagged non-measurement gate?

Reply:
Done